week_2/ec2.tf

```diff
@@ -0,0 +1,22 @@
+variable "instance_name" {
+  description = "Value of the Name tag for the EC2 instance"
+  type        = string
+  default     = "ExampleAppServerInstance"
+}
+
+resource "aws_instance" "app_server" {
+  ami                    = "ami-03ededff12e34e59e"
+  instance_type          = "t2.micro"
```

Comment:
Although not strictly required, these hardcoded values should be also extracted to variables.
Also, variables should be in a separate file, e.g. `variables.tf`

Reply:
added